tests/test_datasets/test_dataset_functions.py

```diff
@@ -1316,3 +1316,18 @@ def test_list_qualities(self):
         qualities = openml.datasets.list_qualities()
         self.assertEqual(isinstance(qualities, list), True)
         self.assertEqual(all([isinstance(q, str) for q in qualities]), True)
+
+    def test_get_dataset_cache_format_pickle(self):
+        # Feather format cant be tested without installing pyarrow
```

Comment:
Could you please add unit tests for feather?

Reply:
We are not adding pyarrow in our installation. So, I am not sure how to test feather

> Could you please add unit tests for feather?

We are not adding pyarrow in our installation. So, I am not sure how to test feather. Do you have some suggestion for this?